js/game.js

```diff
@@ -0,0 +1,154 @@
+// Square Enemies:
+class Game {
+    constructor(ctx, width, height, player) {
+        this.ctx = ctx;
+        this.width = width;
+        this.height = height;
+        this.player = player;
+        this.interval = null;
+        this.frames = 0;
+        this.enemies = [];
+
+    }
+    start() {
+        this.intervalId = setInterval(this.update, 1000 / 60) 
+    }
+    // Update needs to be an arrow function because "this" needs to refer to 
+    // the class and not the update method
+    update = () => {
+        // Game logic here
+        this.frames++;
+        this.clear();
+        this.player.newPos();
+        this.player.draw();
+        this.updateEnemies();
+        this.checkGameOver();
+    }
+    stop() {
+        clearInterval(this.intervalId);
+    }
+    clear() {
+        this.ctx.drawImage(roadImg, 0, 0, canvas.width, canvas.height);
```

Comment:
Here, you're drawing the road image instead of clearing the canvas.
You should've cleared the canvas here and created a separated function or draw the image on the update function